clients/flash/air-client/src/org/bigbluebutton/air/poll/views/PollPopUpMediator.as

```diff
@@ -24,25 +26,30 @@ package org.bigbluebutton.air.poll.views {
 		public override function initialize():void {
 			meetingData.polls.pollChangeSignal.add(onPollChange);
 			view.addEventListener(MouseEvent.CLICK, viewMouseEventHandler);
+			
+			view.removeAllElements();
+			view.addButtons(meetingData.polls.getCurrentPoll());
+			view.stage.addEventListener(StageOrientationEvent.ORIENTATION_CHANGE, onStageOrientation);

```

Comment:
Need to remove this listener in the destroy()

Reply:
Done here https://github.com/bigbluebutton/bigbluebutton/pull/5340/commits/4fc4e177b7b5bd6ca1aeb8621d452ea0bd0d2da0#diff-59e749e9b0b453c0f2d6bd269c02116fR62